src/pages/[domain]/metrics/[metricId]/index.tsx

```diff
@@ -1,9 +1,10 @@
 import { api } from "~/utils/api";
 
-export default function MetricsPage() {
+export default function MetricsPage({ metricId = "" }) {
```

Comment:
Rename to MetricsDetailsPage for clarity.

Use getServerSideProps to get the metricId. See ProjectDetailsPage:
https://github.com/gitcoinco/easy-retro-pgf/blob/c45100740dd756812f16eeb9f5f3d1734bab3f67/src/pages/%5Bdomain%5D/projects/%5BprojectId%5D/index.tsx#L29-L31

Reply:
done